package.json

```diff
@@ -7,28 +7,31 @@
   "scripts": {
     "test": "jest",
     "lint": "eslint --ext js --ext jsx src",
-    "plug": "node injectors/index.js inject --no-exit-codes",
-    "unplug": "node injectors/index.js uninject --no-exit-codes",
-    "inject": "node injectors/index.js inject --no-exit-codes",
-    "uninject": "node injectors/index.js uninject --no-exit-codes"
+    "plug": "ts-node injectors/index.ts inject --no-exit-codes",
+    "unplug": "ts-node injectors/index.ts uninject --no-exit-codes",
+    "inject": "ts-node injectors/index.ts inject --no-exit-codes",
+    "uninject": "ts-node injectors/index.ts uninject --no-exit-codes"
   },
   "repository": "https://github.com/replugged-org/replugged.git",
   "author": "aetheryx & Cynthia K. Rey",
   "bugs": {
     "url": "https://github.com/replugged-org/replugged/issues"
   },
   "dependencies": {
+    "@types/node": "^18.6.1",
```

Comment:
Move to devDependencies

Reply:
Sure, I'll just do @types/node (and not typescript or ts-node, which I think is what you were asking to do in the first place,) as we probably want to include TypeScript as a Repliugged compiler in the future (similar to how we handle SCSS or JSX)

Reply:
Keep TypeScript, just switch out ts-node

Reply:
Currently the updater only installs prod dependencies which breaks the build during the update process. We could either:
1. Keep @types/node as a normal dep
2. Have the updater also install dev deps

I prefer #1, as we probably want to have some dependencies that the updater shouldn't install by default, such as eslint.